src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -76,8 +36,9 @@ export const AppContainer = ({ view }: { view?: View }) => {
       paddingSize="none"
     >
       {/* TODO: improve loading state */}
```

Comment:
is this TODO here still needed? do we plan to improve the loading state?

Reply:
Yeah, this will be improved before merge to main. Its currently not that good.